Jenkinsfile

```diff
@@ -247,6 +247,23 @@ def doBuildInDocker(String command) {
   }
 }
 
+def doBuildWindows() {
+    return {
+        node('windows') {
+            getArchive()
+            try {
+              bat "\"${tool 'msbuild'}\" \"Visual Studio\\Realm.vcxproj\" /p:Configuration=Debug /p:Platform=\"Win32\" /p:VisualStudioVersion=14.0"
```

Comment:
Does it work if you build the solution instead of the project and without specifying `VisualStudioVersion`?